pubsub/gcp/pubsub/pubsub.go

```diff
@@ -76,9 +83,46 @@ type WhatNow struct {
 	Type string `json:"type"`
 }
 
+const (
+	cacheRefreshInterval = 5 * time.Hour
```

Comment:
Please be specific with variable names.

```suggestion
	topicCacheRefreshInterval = 5 * time.Hour
```